.github/ISSUE_TEMPLATE/custom_issue.yml

```
@@ -0,0 +1,19 @@
+name: Custom issue
+description: Is your issue not covered by our bug report or feature request template?
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        It is expected that almost every issue on this project is either a feature request or bug report.
+        Please only select this format in exceptional circumstances, otherwise it will be closed.

```

Maybe we should encourage GH discussions here as well

```suggestion
        Please only select this format in exceptional circumstances, otherwise it will be closed.
        Alternatively, consider a [GitHub discussion](https://github.com/nvaccess/nvda/discussions).
```